docs/topics/assembly_building-and-deploying-the-booster-to-openshiftonline.adoc

```diff
@@ -15,7 +15,8 @@ You have two options for executing the {parameter-mission-name} booster on {Open
 * xref:deploying-the-booster-using-launcher-oso_{context}[Using {launcher-oso}]
 * xref:deploying-the-{parameter-mission}-booster-using-the-oc-cli-client_{context}[Using the `oc` CLI client]
 
-There is no functional difference between these methods, choose the one you prefer.
+While both methods use the same `oc` commands to deploy your application, using
+{launcher-oso} provides and automated booster deployment workflow that executes the `oc` commands behind the scenes.
```

Comment:
Since you changed the wording as per my suggestion in `assembly_building-and-deploying-the-booster-to-openshiftlocal.adoc`, can you please do so here too?

Reply:
Thanks, I'll see to it that it is fixed:)